digiwf-task/digiwf-tasklist-service/src/main/java/io/muenchendigital/digiwf/task/service/adapter/out/schema/VariableTaskSchemaTypeResolverAdapter.java

```diff
@@ -0,0 +1,24 @@
+package io.muenchendigital.digiwf.task.service.adapter.out.schema;
+
+import io.holunda.polyflow.view.Task;
+import io.muenchendigital.digiwf.task.TaskSchemaType;
+import io.muenchendigital.digiwf.task.TaskVariables;
+import io.muenchendigital.digiwf.task.service.application.port.out.schema.TaskSchemaTypeResolverPort;
+import org.springframework.stereotype.Component;
+
+@Component
+public class VariableTaskSchemaTypeResolverAdapter implements TaskSchemaTypeResolverPort {
+  @Override
+  public TaskSchemaType apply(Task task) {
+    Object value = task.getPayload().getOrDefault(TaskVariables.TASK_SCHEMA_TYPE.getName(), null);
```

Comment:
Wenn ich einen Prozess starte und anschließend "Meine Aufgaben" aufrufe, dann kommt ein 500er von der Tasklist: `java.lang.IllegalStateException: Unknown value for task schema type null`. So wie's aussieht steht in den Prozessvariablen kein `app_task_schema_type`.

@zambrovski, @dominikhorn93 Muss ich die Variable händisch im Prozess setzen oder ist das ein Bug. Falls man das händisch setzen muss wäre es sinnvoller als default (anstelle von null) das aktuelle Schema Framework zu nehmen. Dann müssen nur legacy Prozess diese Variable setzen.

Reply:
Hat sich geklärt. In der DB waren noch alte Tasks, bei denen die Variable nicht gesetzt war